python/pylibcugraph/pylibcugraph/experimental/__init__.py

```diff
@@ -66,6 +66,9 @@
 from pylibcugraph.node2vec import node2vec
 node2vec = promoted_experimental_warning_wrapper(node2vec)
 
+from pylibcugraph.bfs import bfs
+bfs = promoted_experimental_warning_wrapper(bfs)
+
```

Comment:
Since we never had `bfs` in experimental, we don't need to add a warning that it was promoted, so these lines can be removed.